exportlib.php

```diff
@@ -84,47 +84,41 @@ public function create_file($courseid, $feedbackid) {
 }
 
 /**
- * @author Felix Di Lenarda
- * rankings CSV export class
+ * Export feedback data for an entire feedbacj.
+ *
+ * @package block
+ * @subpackage coursefeedback
+ * @copyright innoCampus, TU Berlin
+ * @license http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
  */
-class rankingexport {
+class ranking_exporter {
     protected $feedback = null;
     protected $questionid = null;
-    protected $seperator = ";";
-
-    private $content = "";
+    protected $csvexportwriter;
 
     public function __construct($feedback = null, $question = null) {
-        global $DB;
-
-        if ($fb = $DB->get_record("block_coursefeedback", array("id" => $feedback))) {
-            $this->feedback = $fb;
-            $this->questionid = $question;
-        } else {
-            print_error("feedbacknotfound", "error");
-            exit(0);
-        }
+        $this->csvexportwriter = new csv_export_writer();
     }
-
-    public function export() {
+    public function create_file($feedbackid, $questionid = 0 )
```

Comment:
Kleinigkeiten, aber ich würde

1) die Leerzeilen zwischen einzelnen Methoden beibehalten,
2) das Leerzeichen vor der schließenden Klammer entfernen und
3) die öffnende geschweifte Klammer der Konsistenz halber ans Ende der Zeile packen.

Reply:
Angepasst in: 133d532afd97506c96e5730ab0acca9161b6b022